backend/api/deployment_helper.py

```diff
@@ -174,6 +175,7 @@ def execute_workflow(
                 hash_values_of_files=hash_values_of_files,
                 timeout=timeout,
                 execution_id=execution_id,
+                queue=CeleryQueue.CELERY_API_DEPLOYMENTS,
```

Comment:
Where are we  initializing the queue used for task pipelines? @muhammad-ali-e

Reply:
Task piplines and ETLs works with no specific Queues , So default queue will be there (`celery`)